app/commands/storage.html

```diff
@@ -62,8 +62,8 @@
 
   <div class="banner">
     <div class="container">
-      <h1>Local Storage</h1>
-      <p>Examples of managing local storage in Cypress, for a full reference of commands, go to <a href="https://on.cypress.io/api" target="_blank">docs.cypress.io</a>
+      <h1>Storage</h1>
+      <p>Examples of managing local and session storage in Cypress, for a full reference of commands, go to <a href="https://on.cypress.io/api" target="_blank">docs.cypress.io</a>
```

Comment:
```suggestion
      <p>Examples of managing localStorage and sessionStorage in Cypress, for a full reference of commands, go to <a href="https://on.cypress.io/api" target="_blank">docs.cypress.io</a>
```

Reply:
Addressed in https://github.com/cypress-io/cypress-example-kitchensink/pull/579/commits/7e58c1d53861c2719a6cb2ae191e526ab990723c